org.xiphos.Xiphos.json

```diff
@@ -44,6 +44,16 @@
             }
         ]
     },
+    {
+        "name": "python3-libxml2",
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://download.gnome.org/sources/libxml2/2.12/libxml2-2.12.4.tar.xz",
+                "sha256": "497360e423cf0bd99eacdb7c6215dea92e6d6e89ee940393c2bae0e77cb9b7d0"
+            }
+        ]
+    },
```

Comment:
This is in the runtime, try bumping python2-libxml2 to 2.11.5 to match the version in SDK https://gitlab.gnome.org/GNOME/libxml2/-/archive/v2.11.5/libxml2-v2.11.5.tar.bz2

Reply:
Done, thanks!